pydantic/type_adapter.py

```diff
@@ -87,9 +87,12 @@ def _get_schema(type_: Any, config_wrapper: _config.ConfigWrapper, parent_depth:
     But at the very least this behavior is _subtly_ different from `BaseModel`'s.
     """
     local_ns = _typing_extra.parent_frame_namespace(parent_depth=parent_depth)
-    global_ns = sys._getframe(max(parent_depth - 1, 1)).f_globals.copy()
-    global_ns.update(local_ns or {})
-    gen = _generate_schema.GenerateSchema(config_wrapper, types_namespace=global_ns, typevars_map={})
+    globalns = sys._getframe(max(parent_depth - 1, 1)).f_globals
```

Comment:
Can we change `local_ns` in the previous line to be `localns` or `parent_namespace`. Minor I know, lol, but it's bothering me that we have an underscore in `local_ns` and not in `globalns` right next to each other.

Reply:
Sydney actually already requested this but I only applied it to `globalns` :facepalm: